### ISSUE TITLE
Error When using MySQL with PAPI

### ISSUE THREAD

**Issue by PS1TD** [Realizedd/TokenManager#42]
Hi. Great plugin you got here, however, I am getting this strange error when using it with MySQL database and PAPI and Redis.
This error only pops up on server startup, sometimes even multiple times. But the plugin itself seems to work fine.

```
[16:51:47] [Craft Scheduler Thread - 0/WARN]: [PlaceholderAPI] Plugin PlaceholderAPI v2.10.9 generated an exception while executing task 250
java.lang.NullPointerException: null
	at me.realized.tokenmanager.data.database.MySQLDatabase.get(MySQLDatabase.java:123) ~[?:?]
	at me.realized.tokenmanager.data.DataManager.get(DataManager.java:84) ~[?:?]
	at me.realized.tokenmanager.TokenManagerPlugin.getTokens(TokenManagerPlugin.java:207) ~[?:?]
	at me.realized.tokenmanager.hook.hooks.VaultHook$TokenManagerEconomy.getBalance(VaultHook.java:60) ~[?:?]
	at com.extendedclip.papi.expansion.vault.VaultEcoHook.getBalance(VaultEcoHook.java:220) ~[?:?]
	at com.extendedclip.papi.expansion.vault.BalTopTask.run(BalTopTask.java:48) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.3.jar:git-Paper-197]
	at org.bukkit.craftbukkit.v1_16_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) ~[patched_1.16.3.jar:git-Paper-197]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.16.3.jar:git-Paper-197]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_265]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_265]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]

```

**Comment by PS1TD** [comment]
I was able to boil it down to   `register-economy: true` setting in config.yml.
When set to false it does not throw an error, but at the same time the economy is not registered within vault so plugins that rely on vault are not working.
When set to true it throws the error periodically but registers with vault.

**Comment by PS1TD** [comment]
It seems that this issue was addressed in the latest commit. Fixed by using the 3.2.6 dev snapshot version from the developer's discord.
